airbyte-webapp/package.json

```diff
@@ -3,17 +3,18 @@
   "version": "0.40.32",
   "private": true,
   "engines": {
-    "node": "16.18.1"
+    "node": "16.18.1",
+    "pnpm": "7.26.2"
   },
   "scripts": {
```

Comment:
There is actually no difference in using `npm start` or `pnpm start`. The only difference is for managing libraries, why we make sure we enforce on CI that no one does that wrong and switched accidentally package managers again. Locally I considered (see also Evan's and my comments on this PR) adding a check, but the problem is that it adds "seconds" to the command, which given that `pnpm i` might often only take 2s is a bit frustrating so I decided to get rid of it. Also in addition, it won't really save you much time, since the `preinstall` command actually runs... after install since npm 7 (yeah I know very intuitive), so you'd anyway need to go through the full install first before we print you a warning that you used the wrong command (which you also will see by suddenly a new lock file appearing). And for any of the run commands it's not really needed to check, since there is no difference, so I felt we don't need to add any local checks for this. But please feel free to share opposing opinions.

Reply:
Makes sense to me! Thanks for explaining - I think the approach you've used here sounds good